ext/MethodAnalysisExt.jl

```diff
@@ -24,4 +24,28 @@ function MethodAnalysis.methodinstances(info::PkgCacheInfo)
     return mis
 end
 
+function PkgCacheInspector.count_internal_specializations(info::PkgCacheInfo)
+    spec_counts = Dict{Module,Int}()
+
+    # Get all method instances from the cache
+    all_mis = methodinstances(info)
+
+    # Count method instances by their defining module
+    for mi in all_mis
+        try
```

Comment:
Can we eliminate the `try`/`catch` and handle the cases that throw an error? It's not immediately obvious to me what could go wrong.